db/migrate/20220815023618_remove_actvity_notifications_tables.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveActvityNotificationsTables < ActiveRecord::Migration[7.0]
+  def change
+    drop_table :notifications
+    drop_table :subscriptions
```

Comment:
Keep tables for now, otherwise data migration will fail

Reply:
@tachyons Done!